src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -934,6 +936,11 @@ internal virtual void InstallPackage(PackageDownloadHandle packageDownloadHandle
                         }
                     }
                     SetPackageState(packageDownloadHandle, installPath);
+                    Analytics.TrackEvent(Actions.Installed, Categories.PackageManagerOperations, $"{packageDownloadHandle?.Name}");
+                    if (SearchCriteria.Length > 0)
+                    {
+                        Analytics.TrackEvent(Actions.Search, Categories.PackageManagerOperations, $"{SearchCriteria}");
+                    }
```

Comment:
hmm two questions:
1. Why do we track search criteria during package install?
2. I do not think tracking search criteria would give our team too much insights so I would actually suggest removing this track, any objection? @Amoursol @mjkkirschner @zeusongit

Reply:
1. exactly, otherwise I think doesn't make sense track every time the search criteria changing  / it only tracks if the searchCriteria (value of the text search is not empty) 2. It's probable tracking in-depth, let me know if it is worth.

Reply:
I agree, makes no sense tracking search criteria inside InstalledPackages.

Reply:
Done